app/src/main/res/layout/activity_recover_wallet.xml

```diff
@@ -55,6 +55,17 @@
         android:textSize="@dimen/text_size_medium"
         android:gravity="top"/>
 
+    <TextView
```

Comment:
The touch area/feedback needs to be improved. If I touch on the `Andvanced Security` text, then the dialog doesn't appear...I have to touch just above the text area for the dialog to appear. Also applies to `activity_show_mnemonic.xml`